RELEASE-NOTE.md

```diff
@@ -1,3 +1,28 @@
+3.5.4 (2021.03.10)
+==================
+- Changes
+    - Change the HttpClient of the test script to apache http components base
```

Comment:
- Release new experimental HttpClient based on apache http5-core.

Reply:
@junoyoon 

I fixed all. thank you for review.